mongodb/Module.md

```diff
@@ -1,37 +1,27 @@
-# Ballerina MongoDB Module
-
-Connects to MongoDB from ballerina 
-
-## Module Overview
-
+## Overview
 The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
```

Comment:
Suggestion: "MongoDB is a document database designed to cater to modern application requirements. It is a scalable and  a flexible solution that inherently supports distributed system design.  Connecting to the MongoDB allows you to perform the MongoDB CRUD operations."

Reply:
Fixed by 89976637b8fed8c9a16842370da109f71b035569.